docs/releases/2.2.txt

```diff
@@ -148,7 +148,10 @@ Email
 File Storage
 ~~~~~~~~~~~~
 
-* ...
+* Added the ability to pass a callable as the storage parameter of a :class:`~django.db.models.FileField`, or
```

Comment:
move this part to 3.0

Reply:
@nikolaik It needs to be taken out of 2.2 release also.